### ISSUE TITLE
White box flashing below the first notification

### ISSUE THREAD

**Issue by Adrianrff** [wasdennnoch/AndroidN-ify#1020]
As the title said, a blank box flahes below the first notification and covers the one below. I started seeing this after enabling experimental features, but after disabling it it's still showing. Maybe this had  happened before and I just hadn't noticed though. 

Video: https://youtu.be/hNzclFgL3W8

Log: 
[xposed_error_20161005_082857.txt](https://github.com/wasdennnoch/AndroidN-ify/files/511150/xposed_error_20161005_082857.txt)

**Comment by wasdennnoch** [comment]
I can't view the video (it says playback failed). Are you talking about the white background of notifications when you swipe them away? If so, that's N.

**Comment by Adrianrff** [comment]
No, I'm talking about a white rectangle that flashes just below the action buttons (by below I mean further down the screen, not in depth). I'll try to take a screen shot.

**Comment by Adrianrff** [comment]
There you go... 

![5 de octubre de 2016 85249 gmt-0400](https://cloud.githubusercontent.com/assets/22583533/19113791/b0f484ec-8ab7-11e6-8c8b-53d4c520276b.png)
![5 de octubre de 2016 85249 gmt-0400](https://cloud.githubusercontent.com/assets/22583533/19113798/bddf67b2-8ab7-11e6-9123-48b6daf99356.png)
![5 de octubre de 2016 85226 gmt-0400](https://cloud.githubusercontent.com/assets/22583533/19113818/ca918558-8ab7-11e6-94df-bc951e875e85.png)

**Comment by aviraxp** [comment]
Confirm it on every ROM based on MM.
